app/components/pagination-links.js

```diff
@@ -0,0 +1,48 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { equal } from '@ember/object/computed';
+
+export default Component.extend({
+  classNames: ['pagination-links'],
+
+  page: null,
+  results: null,
+  size: null,
+  onNextPage() {},
+  onPrevPage() {},
+  onSelectPage() {},
+
+  disablePrev: equal('page', 1),
+
+  disableNext: computed('lastPage', 'page', function() {
+    return this.lastPage === this.page;
+  }),
+
+  lastPage: computed('results', 'size', function() {
+    return Math.ceil(this.results.length / this.size);
+  }),
+
+  computedPages: computed('page', 'totalPages', function() {
```

Comment:
Just `pages` please. calling it `computedPages` is redundant.

Reply:
👍